python-qa-tools/stats/stats.py

```diff
@@ -0,0 +1,119 @@
+import json, logging
+from db import db
+
+logging.basicConfig(format="%(message)s", level=logging.DEBUG)
+
+COL_NAME = "test_stats"
+TX_TYPE = "tx"
+QUERY_TYPE = "query"
+
+
+def insert_stat(data):
+    _ = db[COL_NAME].insert_one(data)
+
+
+def check_status(test_type, cmd_type, output, err):
```

Comment:
can we rename it to better naming. How about `track` or `record` or maybe `record_stat`?